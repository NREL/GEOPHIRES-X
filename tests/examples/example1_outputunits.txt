Reservoir Model,1,                        			---Multiple, Fractures, reservoir, model
Reservoir Depth,3,                     				---[km]
Number of Segments,1,                  				---[-]
Gradient 1,50,                          			---[deg.C/km]
Maximum Temperature,400,                  			---[deg.C]
Number of Production Wells,2,            			---[-]
Number of Injection Wells,2,            			---[-]
Production Well Diameter,7,		     			---[inch]
Injection Well Diameter,7,					---[inch]
Ramey Production Wellbore Model,1,       			---
Production Wellbore Temperature Drop,.5,			---[deg.C]
Injection Wellbore Temperature Gain,0,   			---[deg.C]
Production Flow Rate per Well,55,       			---[kg/s]
Fracture Shape,3,                       			---[-] 
Fracture Height,900, 						---[m]
Reservoir Volume Option,3,              			---[-]
Reservoir Volume,1000000000,		 			---[m^3]
Number of Fractures,20,		  				---[-]
Water Loss Fraction,.02,					---[-]
Productivity Index,5,						---[kg/s/bar]
Injectivity Index,5,						---[kg/s/bar]
Injection Temperature,50,		 			---[deg.C]
Maximum Drawdown,1,			  			---[-] no redrilling considered
Reservoir Heat Capacity,1000,		  			---[J/kg/K]
Reservoir Density,2700,			  			---[kg/m^3]
Reservoir Thermal Conductivity,2.7,	  			---[W/m/K]

***SURFACE TECHNICAL PARAMETERS***
**********************************
End-Use Option,1,			  			---[-] Electricity
Economic Model,1,			  			---[-] Fixed Charge Rate Model
Power Plant Type,2,			  			---[-] Supercritcal ORC
Circulation Pump Efficiency,.8,	  				---[-] between .1 and 1
Utilization Factor,.9,			  			---[-] between .1 and 1
Surface Temperature,20,		  				---[deg.C]
Ambient Temperature,20,		 				---[deg.C]

***FINANCIAL PARAMETERS***
**************************
Plant Lifetime,30,			  			---[years]
Fixed Charge Rate,.05,			 			---[-] between 0 and 1
Inflation Rate During Construction,0,   			---[-] 

***Simulation Parameters***
***************************

Print Output to Console,1,		  			---[-] Should be 0 (don't print results) or 1 (print results)
Time steps per year,6,		  				---[1/year]

***Output unit conversions you wish to make***
***************************
Units:Bottom-hole temperature, degF,  ---[This is what I want the units to be for this output parameter
Units:Exploration cost,MEUR,  ---[This is what I want the units to be for this output parameter
Units:O&M Make-up Water costs, MEUR/yr
