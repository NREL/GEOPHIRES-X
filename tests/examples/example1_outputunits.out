                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.6.0
 Simulation Date: 2024-10-15
 Simulation Time:  13:07
 Calculation Time:      0.831 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     5.39 MW
      Electricity breakeven price:                            8.71 cents/kWh
      Number of production wells:                             2
      Number of injection wells:                              2
      Flowrate per production well:                          55.0 kg/sec
      Well depth (or total length, if not vertical):          3.0 kilometer
      Geothermal gradient:                                     50 degC/km


                           ***ECONOMIC PARAMETERS***

      Economic Model = Fixed Charge Rate (FCR)
      Fixed Charge Rate (FCR):                                5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                         -34.61 MUSD
      Project IRR:                                          -2.77 %
      Project VIR=PI=PIR:                                    0.26
      Project MOIC:                                         -0.20
      Project Payback Period:                          N/A
      Estimated Jobs Created:                                 12

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             2
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          3.0 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 50.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               3.0 degC
      Flowrate per production well:                          55.0 kg/sec
      Injection well casing ID:                               7.000 in
      Production well casing ID:                              7.000 in
      Number of times redrilling:                             0
      Power plant type:                                       Supercritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                        50 degC/km


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Multiple Parallel Fractures Model
      Bottom-hole temperature:                              338.00 degF
      Fracture model = Square
      Well separation: fracture height:                     900.00 meter
      Fracture area:                                     810000.00 m**2
      Reservoir volume:                              1000000000 m**3
      Reservoir hydrostatic pressure:                         28841.05 kPa
      Plant outlet pressure:                               1067.94 kPa
      Production wellhead pressure:                        1136.89 kPa
      Productivity Index:                                     5.00 kg/sec/bar
      Injectivity Index:                                      5.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir thermal conductivity:                         2.70 W/m/K
      Reservoir heat capacity:                             1000.00 J/kg/K


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       167.3 degC
      Average Production Temperature:                       167.0 degC
      Minimum Production Temperature:                       165.3 degC
      Initial Production Temperature:                       165.3 degC
      Average Reservoir Heat Extraction:                     53.52 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               3.0 degC
      Average Injection Well Pump Pressure Drop:            217.9 kPa
      Average Production Well Pump Pressure Drop:          1112.0 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      16.42 MUSD
         Drilling and completion costs per well:              4.11 MUSD
         Stimulation costs:                                   3.02 MUSD
         Surface power plant costs:                          20.80 MUSD
         Field gathering system costs:                        2.30 MUSD
         Total surface equipment costs:                      23.10 MUSD
         Exploration costs:                                   4.31 MUSD
      Total capital costs:                                   46.85 MUSD
      Annualized capital costs:                               2.34 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.38 MUSD/yr
         Power plant maintenance costs:                       0.90 MUSD/yr
         Water costs:                                         0.06 MUSD/yr
      Total operating and maintenance costs:                  1.34 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.11 MW/(kg/s)
      Maximum Total Electricity Generation:                   5.62 MW
      Average Total Electricity Generation:                   5.59 MW
      Minimum Total Electricity Generation:                   5.42 MW
      Initial Total Electricity Generation:                   5.42 MW
      Maximum Net Electricity Generation:                     5.42 MW
      Average Net Electricity Generation:                     5.39 MW
      Minimum Net Electricity Generation:                     5.22 MW
      Initial Net Electricity Generation:                     5.22 MW
      Average Annual Total Electricity Generation:           43.83 GWh
      Average Annual Net Electricity Generation:             42.28 GWh
      Initial pumping power/net installed power:              3.82 %
      Average Pumping Power:                                  0.20 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                165.34               0.1993            5.2230                9.9020
   2           1.0056                166.27               0.1984            5.3158                9.9972
   3           1.0073                166.54               0.1981            5.3426               10.0245
   4           1.0081                166.68               0.1980            5.3566               10.0387
   5           1.0087                166.77               0.1979            5.3658               10.0481
   6           1.0091                166.84               0.1978            5.3727               10.0551
   7           1.0094                166.89               0.1978            5.3781               10.0605
   8           1.0097                166.94               0.1977            5.3825               10.0650
   9           1.0099                166.97               0.1977            5.3862               10.0688
  10           1.0101                167.01               0.1977            5.3894               10.0720
  11           1.0102                167.03               0.1976            5.3922               10.0749
  12           1.0104                167.06               0.1976            5.3947               10.0774
  13           1.0105                167.08               0.1976            5.3970               10.0797
  14           1.0107                167.10               0.1976            5.3990               10.0817
  15           1.0108                167.12               0.1976            5.4009               10.0836
  16           1.0109                167.14               0.1975            5.4026               10.0853
  17           1.0110                167.15               0.1975            5.4042               10.0869
  18           1.0111                167.17               0.1975            5.4056               10.0884
  19           1.0111                167.18               0.1975            5.4070               10.0898
  20           1.0112                167.19               0.1975            5.4083               10.0911
  21           1.0113                167.21               0.1975            5.4095               10.0923
  22           1.0114                167.22               0.1975            5.4106               10.0935
  23           1.0114                167.23               0.1974            5.4117               10.0946
  24           1.0115                167.24               0.1974            5.4127               10.0956
  25           1.0115                167.25               0.1974            5.4137               10.0966
  26           1.0116                167.26               0.1974            5.4146               10.0975
  27           1.0116                167.27               0.1974            5.4155               10.0984
  28           1.0117                167.27               0.1974            5.4163               10.0993
  29           1.0117                167.28               0.1974            5.4171               10.1000
  30           1.0118                167.29               0.1974            5.4179               10.1008


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  41.6                       417.7                322.50                   0.46
   2                  42.0                       419.8                320.99                   0.93
   3                  42.2                       420.5                319.47                   1.40
   4                  42.3                       420.9                317.96                   1.87
   5                  42.3                       421.1                316.44                   2.33
   6                  42.4                       421.4                314.92                   2.80
   7                  42.4                       421.5                313.41                   3.27
   8                  42.5                       421.7                311.89                   3.74
   9                  42.5                       421.8                310.37                   4.21
  10                  42.5                       421.9                308.85                   4.68
  11                  42.5                       422.0                307.33                   5.14
  12                  42.5                       422.1                305.81                   5.61
  13                  42.6                       422.2                304.29                   6.08
  14                  42.6                       422.2                302.77                   6.55
  15                  42.6                       422.3                301.25                   7.02
  16                  42.6                       422.4                299.73                   7.49
  17                  42.6                       422.4                298.21                   7.96
  18                  42.6                       422.5                296.69                   8.43
  19                  42.6                       422.5                295.17                   8.90
  20                  42.6                       422.6                293.65                   9.37
  21                  42.7                       422.6                292.13                   9.84
  22                  42.7                       422.6                290.60                  10.31
  23                  42.7                       422.7                289.08                  10.78
  24                  42.7                       422.7                287.56                  11.25
  25                  42.7                       422.8                286.04                  11.72
  26                  42.7                       422.8                284.52                  12.19
  27                  42.7                       422.8                282.99                  12.66
  28                  42.7                       422.8                281.47                  13.13
  29                  42.7                       422.9                279.95                  13.60
  30                  35.6                       352.4                278.68                  13.99


                             ********************************
                             *  REVENUE & CASHFLOW PROFILE  *
                             ********************************
Year            Electricity             |            Heat                  |           Cooling                 |         Carbon                    |          Project
Since     Price   Ann. Rev.  Cumm. Rev. |   Price   Ann. Rev.   Cumm. Rev. |  Price   Ann. Rev.   Cumm. Rev.   |   Price   Ann. Rev.   Cumm. Rev.  | OPEX    Net Rev.      Net Cashflow
Start    (cents/kWh)(MUSD/yr) (MUSD)    |(cents/kWh) (MUSD/yr)    (MUSD)   |(cents/kWh) (MUSD/yr)    (MUSD)    |(USD/tonne) (MUSD/yr)    (MUSD)    |(MUSD/yr) (MUSD/yr)    (MUSD)
________________________________________________________________________________________________________________________________________________________________________________________
  1      0.00          -46.85   0.00     |    0.00     0.00         0.00    |    0.00     0.00         0.00     |    0.00     0.00         0.00     |  0.00     -46.85     -46.85
  2      5.50           0.95   2.29     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.95     -45.91
  3      5.50           0.97   4.60     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.97     -44.94
  4      5.50           0.98   6.92     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.98     -43.96
  5      5.50           0.98   9.24     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.98     -42.97
  6      5.50           0.99  11.57     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.99     -41.99
  7      5.50           0.99  13.90     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.99     -41.00
  8      5.50           0.99  16.24     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.99     -40.00
  9      5.50           0.99  18.57     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      0.99     -39.01
 10      5.50           1.00  20.91     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -38.01
 11      5.50           1.00  23.24     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -37.02
 12      5.50           1.00  25.58     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -36.02
 13      5.50           1.00  27.92     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -35.02
 14      5.50           1.00  30.26     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -34.02
 15      5.50           1.00  32.61     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -33.02
 16      5.50           1.00  34.95     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -32.02
 17      5.50           1.00  37.29     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -31.01
 18      5.50           1.00  39.63     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -30.01
 19      5.50           1.00  41.98     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -29.01
 20      5.50           1.00  44.32     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -28.00
 21      5.50           1.00  46.67     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.00     -27.00
 22      5.50           1.01  49.01     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -25.99
 23      5.50           1.01  51.36     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -24.99
 24      5.50           1.01  53.71     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -23.98
 25      5.50           1.01  56.06     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -22.97
 26      5.50           1.01  58.40     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -21.97
 27      5.50           1.01  60.75     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -20.96
 28      5.50           1.01  63.10     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -19.95
 29      5.50           1.01  65.45     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -18.94
 30      5.50           1.01  67.80     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.34      1.01     -17.93
